test/TestInfrastructure/TestExtensions/TestDefaultConfiguration.cs

```diff
@@ -48,6 +52,39 @@ public static void ConfigureHostConfiguration(IConfigurationBuilder builder)
             builder.AddEnvironmentVariables("Orleans");
         }
 
+        /// <summary>
+        /// Hack, allowing PhysicalFileProvider to be serialized using json
+        /// </summary>
+        [Serializable]
+        [JsonObject(MemberSerialization.OptIn)]
+        private class MyPhysicalFileProvider : IFileProvider
```

Comment:
nit: I'd call it something more explicit, such as `SerializablePhysicalFileProvider`